src/pages/gunwoong/Main/mainFooterData.js

```diff
@@ -0,0 +1,9 @@
+const ASIDE_INFO = [
```

Comment:
```suggestion
export const ASIDE_INFO = [
```
아래 export default ASIDE_INFO 를 따로 써주는대신 export를 앞에 붙여 사용하는것이 좋을것같습니다!